packages/tea-map-cache/nyc.config.js

```diff
@@ -2,5 +2,6 @@
 
 module.exports = {
   extends: '../../nyc.config.js',
-  include: ['*.js']
+  include: ['dist'],
+  'exclude-after-remap': false
```

Comment:
I don't think this is needed here. Marc pointed this out on another PR where I had just copy/pasted it

Reply:
It is - if you run test:coverage without it, there appears to be no coverage report.